src/javascript/binary/common_functions/account_opening.js

```diff
@@ -182,12 +182,36 @@ const AccountOpening = (() => {
         return validations;
     };
 
+    const handleAccountType = (e = {}) => {
```

Comment:
do these functions need to be here if they are only used in real_acc_opening.js?